frontend/apps/reader/skimtowidget.lua

```diff
@@ -37,12 +37,12 @@ function SkimToWidget:init()
     self.screen_width = Screen:getWidth()
     self.screen_height = Screen:getHeight()
     self.span = math.ceil(self.screen_height * 0.01)
-    self.width = self.screen_width * 0.95
+    self.width = math.floor(self.screen_width * 0.95)
     self.button_bordersize = Size.border.button
     -- the buttons need some kind of separation but maybe I should just implement
     -- margin_left and margin_right…
     self.button_margin = self.button_bordersize
-    self.button_width = self.screen_width * 0.16 - (2*self.button_margin)
+    self.button_width = math.floor(self.screen_width * 0.16 - (2*self.button_margin))
```

Comment:
math.floor **(** self.screen_width * 0.16 **)** - (2*self.button_margin)
may be? (you did it that way below, which looks clearer)

Reply:
Right, I tried to keep as much precision as possible (so there may be a few cases where I only floor the complete equation *on purpose*), but I indeed missed that one!